code/datums/part_customization.dm

```diff
@@ -52,103 +52,232 @@ ABSTRACT_TYPE(/datum/part_customization/human)
+				if(human.limbs.l_arm?.type != limb_type)
+					human.limbs.replace_with(src.slot, limb_type, null, FALSE, TRUE)
+
+		default_right
+			id = "arm_default_right"
+			slot = "r_arm"
+			part_type = /obj/item/parts/human_parts/arm/right
+
+			apply_to(mob/living/carbon/human/human)
+				var/limb_type = human.mutantrace.r_limb_arm_type_mutantrace
+				if (human.gender == FEMALE) //gendered limbs???
+					limb_type = human.mutantrace.r_limb_arm_type_mutantrace_f || limb_type
+				if (!limb_type)
+					limb_type = src.part_type
+				if(!human.limbs.r_arm?.type == limb_type)
+					human.limbs.replace_with(src.slot, limb_type, null, FALSE, TRUE)
+
+		robo_left
+			id = "arm_robo_left"
+			slot = "l_arm"
+			part_type = /obj/item/parts/robot_parts/arm/left/light
+
+		robo_right
+			id = "arm_robo_right"
+			slot = "r_arm"
+			part_type = /obj/item/parts/robot_parts/arm/right/light
+
+		robo_standard_left
+			id = "arm_robo_standard_left"
+			slot = "l_arm"
+			part_type = /obj/item/parts/robot_parts/arm/left/standard
+			trait_cost = 1
+			incompatible_parts = list("arm_robo_standard_right")
+
+		robo_standard_right
+			id = "arm_robo_standard_right"
+			slot = "r_arm"
+			part_type = /obj/item/parts/robot_parts/arm/right/standard
+			trait_cost = 1
+			incompatible_parts = list("arm_robo_standard_left")
+
+		plant_left
+			id = "arm_plant_left"
+			slot = "l_arm"
+			trait_cost = 1
+			part_type = list(/obj/item/parts/human_parts/arm/left/synth/bloom, /obj/item/parts/human_parts/arm/left/synth)
+
+		plant_right
+			id = "arm_plant_right"
+			slot = "r_arm"
+			trait_cost = 1
+			part_type = list(/obj/item/parts/human_parts/arm/right/synth/bloom, /obj/item/parts/human_parts/arm/right/synth)
+
+	leg
+		default_left
+			id = "leg_default_left"
+			slot = "l_leg"
+			part_type = /obj/item/parts/human_parts/leg/left
+
+			apply_to(mob/living/carbon/human/human)
+				var/limb_type = human.mutantrace.l_limb_leg_type_mutantrace
+				if (human.gender == FEMALE)
+					limb_type = human.mutantrace.l_limb_leg_type_mutantrace_f || limb_type
+				if (!limb_type)
+					limb_type = src.part_type
+				if(human.limbs.l_leg?.type != limb_type)
+					human.limbs.replace_with(src.slot, limb_type, null, FALSE, TRUE)
+
+		default_right
+			id = "leg_default_right"
+			slot = "r_leg"
+			part_type = /obj/item/parts/human_parts/leg/right
+
+			apply_to(mob/living/carbon/human/human)
+				var/limb_type = human.mutantrace.r_limb_leg_type_mutantrace
+				if (human.gender == FEMALE)
+					limb_type = human.mutantrace.r_limb_leg_type_mutantrace_f || limb_type
+				if (!limb_type)
+					limb_type = src.part_type
+				if(human.limbs.r_leg?.type != limb_type)
+					human.limbs.replace_with(src.slot, limb_type, null, FALSE, TRUE)
+
+		robo_left
+			id = "leg_robo_left"
+			slot = "l_leg"
+			part_type = /obj/item/parts/robot_parts/leg/left/light
+
+		robo_right
+			id = "leg_robo_right"
+			slot = "r_leg"
+			part_type = /obj/item/parts/robot_parts/leg/right/light
```

Comment:
No on robolegs here, especially for free. These have a movement modifier that grants 25 points of health slowdown adjustment each.

Reply:
Im removing them.